ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml

```diff
@@ -191,6 +195,26 @@ tests:
     env:
       PROJECT_NAME: numaresources-operator
     workflow: openshift-ci-security
+- as: e2e-operator-upgrade
+  steps:
+    cluster_profile: aws-telco
+    dependencies:
+      OO_BUNDLE: numaresources-operator-bundle
+      OO_BUNDLE_OLD: numaresources-operator-bundle-4.17
+    env:
+      BASE_DOMAIN: telco5g-ci.devcluster.openshift.com
+    test:
+    - ref: telco5g-nrop-install
```

Comment:
We don't need this ref.
We can use `optional-operators-ci-operator-sdk-aws` workflow instead.
The `optional-operators-ci-operator-sdk-aws` installs the operator for us.